### DISCUSSION TITLE
QField elevation receives Ellipsoidal elevation only from Trimble Catalyst DA1 via USB Connection - Orthometric not possible

### DISCUSSION THREAD

**Discussion by jeanbeeckman** [opengisch/QField#4013] [Q&A]
Hello,

Does anyone know how to get orthometric height on QIELD cloud with a GNSS connected via USB cable. The receiver is a Trimble Catalyst DA1. The vertical grid used (.byn format) doesn't change anything because it's always the elipsoidal height collected in may case
Same post on Stackexchange : 
[https://gis.stackexchange.com/questions/414577/qfield-elevation-receives-ellipsoidal-elevation-only-from-trimble-catalyst-da1-v](url)

Thanks for your help

Jean

**Comment by lucienicolier** [comment]
Hi Jean,

QField was improved using external GNSS receiver those last release.

Did you have a look at the documentation [Using an external GNSS Receiver](https://docs.qfield.org/how-to/gnss/?h=gnss#using-an-external-gnss-receiver)

Maybe you will find your answer in here [Altitude correction / vertical grid shift](https://docs.qfield.org/how-to/gnss/?h=gtx#altitude-correction-vertical-grid-shift)

Please tell me if it helps.

Best,

Lucie

**Comment by Bernardodufutur65** [comment]
Hello,

Did someone have found the way to add other grid elevation in Qfield ?

Thanks
